quickwit/quickwit-search/src/root.rs

```diff
@@ -1061,7 +1066,10 @@ pub async fn root_list_terms(
         merged_iter.collect()
     };
 
-    debug!(leaf_list_terms_response = ?leaf_list_terms_response, "Merged leaf search response.");
+    debug!(
+        leaf_list_terms_response_count = leaf_list_terms_response.len(),
+        "Merged leaf search response."
+    );
```

Comment:
@trinity-1686a, as you initially wrote this, are you fine with this lightening of the debug logging ? (`leaf_list_terms_response` seems to be binary, so it should be pretty hard to use anyway 😄)

Reply:
I believe I copied that line from higher up in the file (the other part of that file getting modified). I'm not sure I even used it once. Feel free to do what you wish